Sources/Core/Log/LogOutputStream.swift

```diff
@@ -37,6 +40,7 @@
                 }
                 _ = output.write(bytes, maxLength: data.count)
             }
+#endif
```

Comment:
[imo]

It is preferable to align the indentation with the other lines:

```suggestion
            #endif
```